com.adsbynimbus.nimbus/Editor/NimbusSDKDependencies.xml

```diff
@@ -0,0 +1,86 @@
+    -->
+    <!-- <androidPackage spec="com.google.android.gms:play-services-games:9.8.0">
+  androidSdkPackageIds is optional and contains the set of Android
+         SDK package manager components (packages) this package requires.
+
+         If the package can't be found in any of the specified repositories,
+         the dependency resolver will attempt to install the set of Android
+         SDK packages specified here and then attempt resolution again.
+ 
+      <androidSdkPackageIds>
+        <androidSdkPackageId>extra-google-m2repository</androidSdkPackageId>
+      </androidSdkPackageIds>
+      <repositories>
+        <repository>https://maven.google.com</repository>
+      </repositories>
+    </androidPackage>-->
+  </androidPackages>
+
+  <!-- iOS Cocoapod dependencies can be specified by each iosPod element. -->
+  <iosPods>
+    <!-- iosPod supports the following attributes:
+         * "name" (required)
+           Name of the Cocoapod.
+         * "path" (optional)
+           Path to the local Cocoapod.
+           NOTE: This is expanded to a local path when the Podfile is generated.
+           For example, if a Unity project has the root path "/my/game" and the
+           pod the path is "foo/bar", this will be will be expanded to
+           "/my/game/foo/bar" when the Podfile is generated.
+         * "version" (optional)
+           Cocoapod version specification for the named pod.
+           If this is not specified the latest version is used.
+           NOTE: This can't be used when "path" is set.
+         * "bitcodeEnabled" (optional)
+           Whether this Cocoapod requires bitcode to be enabled in Unity's
+           generated Xcode project.  This is "true" by default.
+         * "minTargetSdk" (optional)
+           The minimum iOS SDK required by this Cocoapod.
+         * "addToAllTargets" (optional)
+           Whether to add this pod to all targets when multiple target is
+           supported. This is "false" by default.
+         * "configurations" (optional)
+           Podfile formatted list of configurations to include this pod in.
+         * "modular_headers" (optional)
+           Set to true to enable modular headers, false to disable.
+         * "source" (optional)
+           Source repo to fetch the pod from.
+         * "subspecs" (optional)
+           Subspecs to include for the pod.
+     -->
+    <!--<iosPod name="NimbusSDK" version="~> 2.20.0" addToAllTargets="true"
+        subspecs="['NimbusKit', 'NimbusRenderVideoKit', 'NimbusRenderStaticKit']">
+      <sources>
+        <source>https://github.com/CocoaPods/Specs</source>
+      </sources>
+    </iosPod>-->
+    <!--The below pod is only to trick EDM4U into building a podfile but this is then removed and replaced dynamically
+      with the actual needed pods (NimbusSDK + related) -->
+    <iosPod name="Google-Mobile-Ads-SDK" version="~> 7.0" bitcodeEnabled="true"
```

Comment:
1. This trick here is suspicious; I was under the impression that EDM4U should completely handle importing the dependency and our PostBuildProcessor would not be used in that case.
2. Any dependencies should be defined as bitcodeEnabled="false" due to changes in Xcode 16.

Reply:
So the issues is that we dynamically ad APS dependencies and add SkaAdNetworkIds to the PLIST dynamically.  Their documentation references using the postBuildProcessor if you want to add things dynamically.  Also, I had to put this "placeholder" in there so it would create a Podfile.